src/pet.js

```diff
@@ -0,0 +1,22 @@
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+}
+Pet.prototype.growUp = function () { 
```

Comment:
Should add a clear line to separate the start of this function from the end of the last one